restaurant/templates/restaurant/category_add.html

```diff
@@ -0,0 +1,118 @@
+{% extends 'restaurant/base.html' %}
+{% load static %}
+
+{% block content %}
+      <div class="col-md-9 ml-auto">
+        <div class="row">
+          <div class="right__body__sec">
+            <div class="pd-tb-md">
+              <div class="rb-pd-rl rs-in-hl">
+                <div class="rs-in">
+                  <div class="rs-lg">
+                    <img src="{% static 'restaurant/images/rest_logo.jpg' %}" alt="" />
+                  </div>
+                  <div class="rs-in-tx">
+                    <div class="rs-nm">Name of the Restarurant</div>
```

Comment:
Better if these are kept on the `base.html` file. Also, the name of restaurant and address of restaurant needs to be dynamic. The restaurant detail can be obtained using request.restaurant.